components/Spotlight/Spotlight.module.css

```diff
@@ -0,0 +1,155 @@
+.MainSpotlightWrapper{
+    height: auto;
+    width: 100%;
+}
+
+.MainSpotlight{
+    margin-top: 100px;
+    margin-right: 10px;
+    position: relative;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    max-height: 500px;
+    width: 100%;
+}
+
+.MainCard{
+    max-height: inherit;
+    width: 50%;
+    position: relative;
+}
+
+.MainCardTitle{
+    margin-top: -10px;
```

Comment:
I generally try to avoid negative margin values like the plague — they make for some tricky layout troubleshooting down the line.

What are you trying to achieve here? It looks like decreasing the vertical space above the title? I'd prefer we make the header narrower instead if this is the desired outcome.

Reply:
Will keep that in mind, I'll make the change to take out negative margin vals. I think I was having trouble because there were some conflicting css elements and styles from the modernism css or from other elements we had from the navigation bar. It was just to bring the title a little bit higher for looks. Again, I can definitely remove it and debug from the element above or adjust the other elements accordingly